Add docs updates for Azure provisioning using custom images (#258)
theforeman/foreman-documentation: guides/common/modules/proc_adding-azure-connection.adoc

. In the *Client Secret* field, enter the client secret.
. In the *Subscription ID* field, enter the subscription ID.
. In the *Tenant ID* field, enter the Directory or _tenant_ ID.

> . Click *Load Regions*. This tests if your connection to the Azure Resource Manager was successful, and accordingly will load the available regions specific to your subscription.

Reviewer: ```suggestion
. Click *Load Regions*. This tests if your connection to Azure Resource Manager is successful and loads the regions available in your subscription.
```
Keep things in present tense for simplicity. Remove article before a product name. Remove fluff.

Author: Done
